website/src/content/docs/pt-br/formatter/index.mdx

```diff
@@ -1,45 +1,45 @@
 ---
-title: Formatter
-description: How to use the Biome formatter.
+title: Formatador
+description: Como usar o formatador do Biome.
 ---
 
 import PackageManagerBiomeCommand from "@src/components/PackageManagerBiomeCommand.astro";
 
-Biome is an opinionated formatter that has the goal to stop all ongoing debates over styles.
-It follows a similar [philosophy to Prettier](https://prettier.io/docs/en/option-philosophy.html),
-only supporting a few options to avoid debates over styles, turning into debates over Biome options.
-It deliberately [resists the urge to add new options](https://github.com/prettier/prettier/issues/40) to prevent [bike-shed discussions](https://en.wikipedia.org/wiki/Law_of_triviality) in teams so they can focus on what really matters instead.
+O Biome é uma ferramenta de formatação com convicções fortes que tem o objetivo de acabar com os contínuos debates sobre estilo de código.
+Ele segue uma [filosofia semelhante à do Prettier](https://prettier.io/docs/en/option-philosophy.html) (link em inglês),
+dando suporte a apenas algumas opções para evitar que debates sobre estilos se tornem debates sobre as opções de configuração do Biome.
+Ele deliberadamente [resiste ao afã de adicionar novas opções](https://github.com/prettier/prettier/issues/40) para evitar [discussões triviais](https://pt.wikipedia.org/wiki/Lei_da_trivialidade) nas equipes, para que elas possam se concentrar no que realmente importa.
 
-## Options
+## Opções
 
-The language agnostic options supported by Biome are:
+As opções, agnósticas da linguagem, suportadas pelo Biome são:
 
-- indent style (default: `tab`): Use spaces or tabs for indention
-- tab width (default: `2`): The number of spaces per indention level
-- line width (default: `80`): The column width at which Biome wraps code
+- estilo de indentação (padrão: `tab`): Use espaços ou tabulações para a indentação
+- largura da tabulação (padrão: `2`): O número de espaços por nível de indentação
+- comprimento de linha (padrão: `80`): Número de colunas máximo a partir do qual Biome quebra a linha
```

Comment:
```suggestion
- comprimento de linha (padrão: `80`): Largura máxima da coluna a partir do qual Biome quebra a linha
```

Reply:
Isso está errado, não é sobre a largura da coluna, mas sobre o número de colunas. Cada caractér é uma coluna.

Reply:
Ok, Do we need to update the english version too?

Reply:
I don't know about that. I did not write the original documentation, nor am I part of the core contributors. However, this is _**pretty standard**_ language. When talking about files or text editors. A file has lines and columns. A column is the same as a character. I think "column's width" would be unequivocally about the "width of the column", but just "column width" can be ambiguous and mean either one. What I know is that "largura da coluna", as if a line has a single column, is wrong, unless Biome is trying to redefine a standard concept.

Reply:
Too much effort for something so small.

Lets keep your suggestion, thank you for you hard work.